src/main/java/no/unit/nva/cristin/projects/PresentationConverter.java

```diff
@@ -12,7 +12,7 @@ protected ProjectPresentation asProjectPresentation(Project project, String lang
         projectPresentation.cristinProjectId = project.cristinProjectId;
         projectPresentation.mainLanguage = project.mainLanguage;
 
-        Optional.ofNullable(project.title).orElse(new TreeMap<String, String>() {
+        Optional.ofNullable(project.title).orElse(new TreeMap<>() {
```

Comment:
Collections.emptyMap().
refactor to method

Reply:
1, actually not.
2. next PR